patches/server/0361-Implement-alternative-item-despawn-rate.patch

```diff
@@ -4,101 +4,149 @@ Date: Mon, 3 Jun 2019 02:02:39 -0400
 Subject: [PATCH] Implement alternative item-despawn-rate
 
 
+diff --git a/src/main/java/com/destroystokyo/paper/PaperConfig.java b/src/main/java/com/destroystokyo/paper/PaperConfig.java
+index e683e5bf47abe7bd3d2f7e9811a377549308ded4..a3c3656ba4e8bb85bfb3186d6284f02f09975b13 100644
+--- a/src/main/java/com/destroystokyo/paper/PaperConfig.java
++++ b/src/main/java/com/destroystokyo/paper/PaperConfig.java
+@@ -70,8 +70,8 @@ public class PaperConfig {
+         commands = new HashMap<String, Command>();
+         commands.put("paper", new PaperCommand("paper"));
+ 
+-        version = getInt("config-version", 24);
+-        set("config-version", 24);
++        version = getInt("config-version", 25);
```

Comment:
changes to this should be in the OG patch

Reply:
Thanks, fixed.